src/stories/layouts/Wizard/Wizard.js

```diff
@@ -0,0 +1,106 @@
+import './styles.scss';
+
+import React, { Fragment, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import StepWizard from 'react-step-wizard';
+import { Steps } from '../../components/Steps/Steps';
+import { Constrain } from '../../components/Constrain/Constrain';
+
+/**
+ * Component for Wizard layout element.
+ * @param {node} children of the component
+ * @param {array} labels of steps
+ * @return {object} (
+ *   <Wizard labels={labels}>
+ *       {children}
+ *   <Wizard />
+ * )
+ */
+/* eslint react/prop-types: 0 */
+export const Wizard = ({children, labels}) => {
+    const [state, updateState] = useState({
+        form: {},
+    });
+
+    // Do something on step change
+    const onStepChange = (stats) => {
+        // might need later on
+    };
+
+    const setInstance = (SW) => updateState({
+        ...state,
+        SW,
+    });
+
+    const { SW } = state;
+
+    function renderWizard() {
+        return (
+            <Constrain>
+                <StepWizard
+                    className="wizard"
+                    onStepChange={onStepChange}
+                    nav={
+                        <Steps labels={labels} />
+                    }
+                    instance={setInstance}
+                >
+                    {children}
+                </StepWizard>
+                { SW && <Controls SW={SW} /> }
+            </Constrain>
+        );
+    }
+    return (
+        <div>
+            { children.length === labels.length ?
+                renderWizard() :
+                <Constrain modifierClasses="constrain--small">
+                    A message goes here.
```

Comment:
This is a placeholder for now. We should probably create an `<Alert />` component at some point to provide system feedback when needed.